### PR TITLE
Better sig loading

### LINKED ISSUE #667: improve signature handling/loading
- make cmd line arguments more consistent with rules
- better error handling

### PR DESCRIPTION
**Pull request by mr-tz**
closes #667 

### Checklist

- [x] No CHANGELOG update needed

- [ ] No new tests needed

- [ ] No documentation update needed

### CODE REVIEW

File: capa/main.py [hunk 1]
```diff
@@ -698,11 +702,11 @@ def install_common_args(parser, wanted=None):
 
     if "signatures" in wanted:
         parser.add_argument(
-            "--signature",
-            dest="signatures",
+            "-s",
```
Comment (williballenthin) [inline capa/main.py:705]: is this option used commonly enough to warrant a shortcut?

**Review by williballenthin — APPROVED** [review]

File: capa/main.py (hunk 1 above)
Comment (mr-tz) [reply capa/main.py:705]: I've added it for consistency with `--rules` which is likely used as often(?) 😕

Comment (mr-tz) [reply capa/main.py:705]: same for `--backend`?

Comment (williballenthin) [reply capa/main.py:705]: ok, fine with me

**Pull request merged**